app/controllers/github_repos_controller.rb

```diff
@@ -10,7 +10,7 @@ def index
     # NOTE: this will invoke autopaging, by issuing multiple calls to GitHub
     # to fetch all of the user's repositories. This could eventually become slow
     @repos = fetch_repositories_from_github(known_repositories_ids)
-  rescue Octokit::Unauthorized => e
+  rescue Github::Errors::Unauthorized => e
```

Comment:
🔥 I always prefer wrapping dependencies and not directly exposing them to most of the codebase.